Chap_API_Proc_Mgmt.tex

```diff
@@ -536,6 +554,22 @@ \subsection{Spawn attributes}
 \declareAttribute{PMIX_EVENT_SILENT_TERMINATION}{"pmix.evsilentterm"}{bool}{
 Do not generate an event when this job normally terminates.
 }
+%
+\declareAttributeNEW{PMIX_ENVARS_HARVESTED}{"pmix.evar.hvstd"}{bool}{
+Environmental parameters have been harvested by the spawn requestor - the server
+does not need to harvest them.
+}
+%
+\declareAttributeNEW{PMIX_JOB_TIMEOUT}{"pmix.job.time"}{int}{
+Time in seconds before the spawned job should time out and be terminated (0 => infinite)
```

Comment:
Is this a total runtime timeout? Kind of like the walltime limits on batch jobs?

Reply:
I addressed this - please see if the revised text helps.

Reply:
That looks good. Thanks!